project/AkkaBuild.scala

```diff
@@ -213,103 +202,34 @@ object AkkaBuild extends Build {
     dependencies = Seq(persistence % "compile;provided->provided;test->test", testkit % "compile;test->test")
   )
 
-  lazy val streamAndHttp = Project(
```

Comment:
I am not sure it is worth having it and adding additional complexity to the build. Also this project was not solving the problem completely, because one still needs to run `akka-docs/test` and jmh tests separately to see if everything depending on streams compile and run.